avm/res/fabric/capacity/README.md

```diff
@@ -0,0 +1,242 @@
+# Fabric Capacities `[Microsoft.Fabric/capacities]`
+
+<Add description>
```

Comment:
Readme to be regenerate as a last step after addressing other comments

Reply:
Fixed